Source/WebCore/svg/SVGPathElement.cpp

```diff
@@ -192,6 +192,10 @@ ExceptionOr<Ref<SVGPoint>> SVGPathElement::getPointAtLength(float distance) cons
     // Spec: Clamp distance to [0, length].
     distance = clampTo<float>(distance, 0, getTotalLength());
 
+    // Spec: If it is not able to compute the total length of path, then throw.
+    if (pathByteStream().isEmpty())
+        return Exception { ExceptionCode::InvalidStateError, "The element's path is empty."_s };
```

Comment:
Does this need to happen after clamping?

Reply:
Nope - I think we can move this block above as well.

Reply:
Even in spec, clamping is after this, so it make sense to move this above. Will do in next push. I stopped `merge-queue`.